app/views/volunteer_ops/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_for(@volunteer_op, url: submission_url ) do |f| %>
+<%= form_for(@volunteer_op, url: submission_url, html: {method: action}) do |f| %>
```

Comment:
`action` is being pass from `locals` and it will be `post` or `put` depending the view that is calling this partial, I had to specify the method since it was failing and making `posts`.